scripts/main.js

```diff
@@ -380,9 +380,35 @@ var villain_character = 2;
 
 
 
-
-
-
+	function getSkinFromId(playerId) {

```

Comment:
Again, sorry for making you scrap your own work, but it's best not to leave unused code behind. Could you remove this function?